travisci/harness.sh

```diff
@@ -1,18 +1,17 @@
 #!/bin/bash
 
-ENSDIR="${ENSDIR:-$PWD}"
+export PERL5LIB=$PWD/bioperl-live-bioperl-release-1-2-3:$PWD/ensembl-test/modules:$PWD/ensembl/modules:$PWD/modules:$PWD/ensembl-variation/modules:$DEPS/Bio-HTS/blib/lib/:$DEPS/Bio-HTS/blib/arch:$PERL5LIB
```

Comment:
This looks very wrong.

Reply:
indeed... was trying to fix it but gave up for now and reverted to that original state